src/app/InteractionModelEngine.h

```diff
@@ -164,6 +164,17 @@ class InteractionModelEngine : public Messaging::ExchangeDelegate
 
 void DispatchSingleClusterCommand(chip::ClusterId aClusterId, chip::CommandId aCommandId, chip::EndpointId aEndPointId,
                                   chip::TLV::TLVReader & aReader, Command * apCommandObj);
+
+/**
+ *  Check if a cluster command exists, the caller would expect this function to look up a catalog to find if the cluster / command
+ * exists on the endpoint.
```

Comment:
```suggestion
 *  Check whether the given cluster exists on the given endpoint and supports the given command.
```
How does this handle client vs server sides of the cluster, especially given that they can have overlapping command id spaces?

Reply:
Added a comment with spec issue that we cannot fill endpoint id in response command.